packages/documentation/src/stories/modules/header/header.stories.ts

```diff
@@ -42,6 +43,16 @@ const meta: MetaComponent = {
         category: 'Content',
       },
     },
+    targetGroup: {
+      name: 'Target group',
+      description: 'Whether or not the target group buttons are visible.',
+      control: {
+        type: 'boolean',
+      },
+      table: {
+        category: 'Content',
+      },
+    },
```

Comment:
This arg is not used in the render function, so it does not hide/show the target group at all.